.circleci/config.yml

```diff
@@ -38,15 +38,6 @@ references:
         sudo usermod -aG docker ubuntu
         sudo chown -R ubuntu:ubuntu ~/epoch
 
-  install_docker_client: &install_docker_client
-    run:
-      name: Install Docker CLI tools
-      command: |
-        curl -L -o /tmp/docker-${DOCKER_CLIENT_VERSION:?}.tgz \
```

Comment:
Is `DOCKER_CLIENT_VERSION: "17.09.0-ce"` in this file now unused and to be removed?

Reply:
It might be removed. I did not find any other references it in in the .circleci/config.yml. I was not sure. @dincho ?

Reply:
I don't see other references either, let's remove it

Reply:
Fixed in 45e3b44